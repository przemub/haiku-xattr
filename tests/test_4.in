# Multiple attributes are added and printed successfully
	xattr -w test_attr test_val file && 
	xattr -w test_attr2 test_val2 file &&
	xattr -p test_attr file &&
	xattr -p test_attr2 file
