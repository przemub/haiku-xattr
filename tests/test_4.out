test_val
test_val2
